### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by saulshanabrook**

### CODE REVIEW

**Comment by blink1073** [comment]
@saulshanabrook, do you mind adding me as an owner on Azure pipelines?

**Comment by saulshanabrook** [comment]
Closing in favor of https://github.com/jupyterlab/debugger/pull/15

@blink1073 I think you are?

Screen Shot 2019-08-20 at 5 07 20 PM

Should I add a different email?

**Comment by blink1073** [comment]
Let me try again.  Last time I couldn't enable this repo.

**Comment by blink1073** [comment]
Looks like I have access.  I might have tried the wrong thing last time.  Cheers!
